src/cmd/bootstrap.test.ts

```diff
@@ -202,6 +202,11 @@ describe('Bootstrapping values', () => {
       beforeEach(() => {
         deps.isChart = false
       })
+      it('should retrieve previous user input when cluster provider is set', async () => {
+        deps.loadYaml.mockReturnValue({ ...values, cluster: { provider: 'set' } })
+        await processValues(deps)
+        expect(deps.hfValues).toHaveBeenCalledWith({ filesOnly: true })
```

Comment:
why don't you check values that processValues returns?

Reply:
it is not what is being tested, and we just stubbed the user input

Reply:
also, the coverage of the parts in processValues after is made by the rest of the tests. Check the coverage.